client/src/modules/inventory/list/modals/actions.tmpl.html

```diff
@@ -184,6 +184,30 @@
       </label>
     </div>
 
+    <hr>
+
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" name="tracker" ng-true-value="1" ng-false-value="0" ng-model="$ctrl.item.tracker">
+        <span translate>INVENTORY.TRACKER</span>
+      </label>
+    </div>
+
+    <div class="form-group">
+      <label class="control-label" translate>INVENTORY.IMPORTANCE_LEVEL</label>
+      <select
+        class="form-control"
+        name="importance"
+        ng-model="$ctrl.item.importance">
+        <option value="" disabled translate>INVENTORY.IMPORTANCE_LEVEL_SELECT</option>
```

Comment:
In the issue https://github.com/IMA-WorldHealth/bhima/issues/4858, you mention that NULL is the default importance level and it means "do not track".  I propose that you remove the column `tracker` and just have this column with `NULL` being "not tracked".  What do you think?

Reply:
You would then change this from "Select Importance Level" to "do not track inventory item", and then "priority: low", "priority: medium", etc.